mzLib/MassSpectrometry/MzSpectra/SpectralSimilarity.cs

```diff
@@ -173,6 +173,67 @@ private List<(double, double)> IntensityPairs(bool allPeaks)
             return intensityPairs;
         }
 
+        //This function is used for calculating correctedIntensityPairs after renormalization
+        private List<(double, double)> CorrectedIntensityPairs(double[] tempExperimentalY, double[] tempTheoreticalY, bool allPeaks)
```

Comment:
can we just call the original intensity pairs function w/ the revised arrays? i can't tell immediately if this is just duplicated code.

Reply:
Deleted this function